src/Queue/Adapter/Swoole.php

```
@@ -27,13 +31,16 @@ public function start(): self
     {
+        if ($this->onStop) {
+            call_user_func($this->onStop);
+        }
         $this->pool->shutdown();
         return $this;
     }
 
     public function workerStart(callable $callback): self
     {
-        $this->pool->on('WorkerStart', function (Pool $pool, string $workerId) use ($callback) {
+        $this->pool->on(Constant::EVENT_WORKER_START, function (Pool $pool, string $workerId) use ($callback) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$pool'. (Unused Code Rules)

(UnusedFormalParameter)